docs/customising/config.md

```diff
@@ -614,6 +616,39 @@ THEME_URLS:
   </dd>
 
   <dt>
+    <a name="incoming_email_spam_header"><code>INCOMING_EMAIL_SPAM_HEADER</code></a>
```

Comment:
Should add the version number of when this became available https://github.com/mysociety/alaveteli/issues/2177#issuecomment-86464175